deployments/opmainnet_addresses.json

```diff
@@ -2,6 +2,13 @@
   "name": "opmainnet",
   "chainId": "10",
   "addresses": {
+    "DefaultProxyAdmin": "0xA14F7F19738cbCF552974F6a20feA86F553Cc13c",
```

Comment:
Shouldn't we reuse the ProxyAdmin deployed by the `oracles` project (`0xeaF9490cBEA6fF9bA1D23671C39a799CeD0DCED2`)?